.vib/parse/goss/dashboard/goss.yaml

```diff
@@ -0,0 +1,25 @@
+file:
+  /bitnami/parse-dashboard:
+    exists: true
+    filetype: directory
+    mode: "2777"
+    owner: root
+  /opt/bitnami/parse-dashboard/config.json:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - /masterKey.*{{ .Vars.server.masterKey }}/
```

Comment:
A bit trivial, but we could drop this check. We are already checking it on the API request.

Reply:
Actually no, the API request checks the **Server** component is configured with the _masterKey_. This is checking the **Dashboard**, for which we can't implement any Cypress tests at the moment.

This way we ensure both have matching values.

Reply:
My bad then, thanks for the clarification!